src/main/app/drafts/models/claimant.ts

```diff
@@ -1,14 +1,96 @@
-import Person from 'drafts/models/person'
+import { CompanyDetails } from 'forms/models/companyDetails'
+import { SoleTraderDetails } from 'forms/models/soleTraderDetails'
+import { OrganisationDetails } from 'forms/models/organisationDetails'
+import { IndividualDetails } from 'forms/models/individualDetails'
+import { PartyType } from 'forms/models/partyType'
+import { PartyDetails } from 'forms/models/partyDetails'
+import { MobilePhone } from 'app/forms/models/mobilePhone'
 import Payment from 'app/pay/payment'
+import { CompletableTask } from 'app/models/task'
 
-export default class Claimant extends Person {
+export default class Claimant implements CompletableTask {
+  partyDetails?: PartyDetails
+  mobilePhone?: MobilePhone
   payment: Payment = new Payment()
 
-  deserialize (input: any): Claimant {
+  static fromObject (input?: any): Claimant {
```

Comment:
Method `fromObject` is used to deserialise form objects. Method not needed and never used.

ACtION NEEDED

Reply:
Done